README.md

```diff
@@ -845,7 +845,7 @@ transaction = MessageTransactionBuilder()\
     .step_limit(1000000)\
     .nid(3)\
     .nonce(100)\
-    .data("test")\
+    .data("0x00..0")\
```

Comment:
It would be better to use real example like:
```suggestion
    .data("0x74657374")\
```

Reply:
Applied.